libcloudforensics/providers/azure/internal/common.py

```diff
@@ -133,12 +115,106 @@ def GetCredentials(profile_name: Optional[str] = None
           'file should contain at least the following: {0:s}'.format(
               ', '.join(required_entries)), __name__)
 
-    # Set environment variables so that DefaultAzureCredentail can pick them up.
+  return account_info
+
+
+def _CheckAzureCliCredentials() -> Optional[str]:
+  """Test if AzureCliCredentials are configured, returning the subscription
+  id if successful.
+
+  Returns:
+    str: the subscription_id of the credentials if properly configured or else
+      None.
+
+  Raises:
+    CredentialsConfigurationError: If AzureCliCredentials are configured but
+      the active subscription could not be determined.
+  """
+  subscription_id = None
+  tokens = None
+
+  config_dir = os.getenv('AZURE_CONFIG_DIR')
+  if not config_dir:
+    config_dir = os.path.expanduser('~/.azure/')
+
+  tokens_path = os.path.join(config_dir, 'accessTokens.json')
+  profile_path = os.path.join(config_dir, 'azureProfile.json')
+
+  if not os.path.exists(tokens_path) or not os.path.exists(profile_path):
+    return None
+
+  with open(tokens_path, encoding='utf-8-sig') as tokens_fd:
+    tokens = json.load(tokens_fd)
+
+  # If tokens are found then Azure CLI auth should be configured.
+  if tokens:
+    with open(profile_path, encoding='utf-8-sig') as profile_fd:
+      profile = json.load(profile_fd)
+
+      for subscription in profile['subscriptions']:
+        if subscription['isDefault']:
+          subscription_id = subscription["id"]
```

Comment:
```suggestion
          return subscription["id"]
```

and remove the break as well as the return on line 164